src/chainparams.h

```diff
@@ -188,6 +189,8 @@ class CChainParams
     int nMinSporkKeys;
     uint16_t nDefaultPlatformP2PPort;
     uint16_t nDefaultPlatformHTTPPort;
+    /// The number of blocks the credit pool tracks; 576 (one day) on mainnet, reduced on regtest
```

Comment:
Maybe then apply 100 on all test nets? (Trstnet, devnet, regtest?

Reply:
I don't see a point to introduce breaking changes for devnet / testnet over this. Only real benefit is in CI situations imo